### ISSUE TITLE
On archlinux keyart is printing out 1 real and 3 bogus keyart boxes

### ISSUE THREAD

**Issue by snassar** [atoponce/keyart#4]
I am not sure what is happening here but I get the same result whether invoking python2 (2.7) or python3 (3.4)

```
$ python3 keyart -l 0xFE679A908E997AB2
+-----[RSA 4096]----+
|                   |
|                   |
|                   |
|                   |
|                   |
|         S     .   |
|        . .   :    |
|         ^ ^ ^ i   |
|        . . l l l .|
|         ^.:^? ^ i^|
|        . E(? . :^.|
+-[FE679A908E997AB2]+
+-------------------+
|   .:              |
|   . : .   .       |
|  . . ^ ^ ^        |
|   ^ . ^ ^ .       |
|  l ^   ^ ^        |
| . i ^   S         |
|  . i . ^          |
|     l . : .       |
|    . . ^E^.       |
|       ^ .^        |
|        ^.         |
+-------------------+
+-------------------+
|            ^il    |
|           : ^.^  .|
|          . : ( .^.|
|           ^ l l.^^|
|          . ^ :....|
|         S . . .E..|
|                ...|
|                 ..|
|                . .|
|                 ^.|
|                ..^|
+-------------------+
+-------------------+
|   .  ... ^^ ..    |
|    ^ :. .  .  .   |
| E : ^ .    . l    |
|  l .        i ^   |
| . ^    . ^ ^ ^    |
|  ^    . S i . .   |
| .      . : : .    |
|         ^ ^       |
|        .   .      |
|                   |
|                   |
+-------------------+

```

**Comment by atoponce** [comment]
Please paste the output of:

```
$ gpg --with-colons --fingerprint -- 0xFE679A908E997AB2 | grep -E '^(pub|fpr)'
```

**Comment by snassar** [comment]
```
fpr:::::::::EE76B39E07788F95F796B044FE679A908E997AB2:  
fpr:::::::::F081A87844D1C6F7E5EA7C220843DE6EE49FE725:  
fpr:::::::::4593BB955BB7E73F6B0404E78C1C1114D3DF46EA:  
fpr:::::::::3D3E90F85633F5128D6C4D208448DA08EB5A611A:
```

**Comment by atoponce** [comment]
Okay. We found the culprit. It is due to having `with-fingerprint` in your `~/.gnupg/gpg.conf`.

**Comment by atoponce** [comment]
Fixed with commit 711d2ba59259be333cc7e8dcf2d90a00115f8842.
